rulings/cards-rulings.yaml

```diff
@@ -982,6 +984,7 @@
 "100908|Heidelberg Castle, Germany":
     - 'May not be locked "in response" to an effect: all effects are instantaneous except for actions and certain combat effects. [LSJ 20010110] [LSJ 20050111]'
     - "Cannot be used to put a card on a vampire who is prohibited to have it (eg. an equipment on {Enkidu, The Noah}). [LSJ 19980831]"
+    - "Cannot be used before damage is inflicted after action resolution (eg. {Force of will}, {Daring the Dawn}): this is still during the action. [ANK 20171124]"
```

Comment:
Reworded a bit: Heidelberg could be used just before damage is inflicted (eg. [Talbot's Chainsaw](https://codex-of-the-damned.org/card-search/index.html?card=Talbot%27s%20Chainsaw)) if this is not during the action.